data/experiment-recipe-samples/desktop-95-multifeature.json

```diff
@@ -0,0 +1,72 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "branches": [
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {}
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": false
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "control"
+    },
+    {
+      "features": [
+        {
+          "enabled": true,
+          "featureId": "pocketNewtab",
+          "value": {
+            "compactLayout": true,
+            "lastCardMessageEnabled": true,
+            "loadMore": true,
+            "newFooterSection": true
+          }
+        },
+        {
+          "featureId": "upgradeDialog",
+          "value": {
+            "enabled": true
+          }
+        }
+      ],
+      "ratio": 1,
+      "slug": "treatment"
+    }
+  ],
+  "bucketConfig": {
+    "count": 10000,
+    "namespace": "firefox-desktop-multifeature-test",
+    "randomizationUnit": "normandy_id",
+    "start": 0,
+    "total": 10000
+  },
+  "channel": "nightly",
+  "endDate": null,
+  "featureIds": [
+    "upgradeDialog"
+  ],
+  "id": "mr2-upgrade-spotlight-holdback",
+  "isEnrollmentPaused": true,
+  "outcomes": [],
+  "probeSets": [],
+  "proposedDuration": 63,
+  "proposedEnrollment": 7,
+  "referenceBranch": "control",
+  "schemaVersion": "1.6.2",
+  "slug": "firefox-desktop-multifeature-test",
+  "startDate": "2021-10-26",
+  "targeting": "true",
```

Comment:
Should targeting be different for desktop? I remember experiments having at least 'app.shield.optoutstudies.enabled'|preferenceValue. I am not sure if this is significant.

Reply:
No it shouldn't be significant here, I tried to set the recipe up so it would definitely enroll.  Testing the targeting expressions is handled elsewhere, nothing changes for that here.